scripts/functions/generate_data_parallel.py

```diff
@@ -0,0 +1,1143 @@
+'''
+Author: Alexandra Lee
```

Comment:
didn't I review this before? is this new?

Reply:
This file is very similar to ```generate_data.py``` but the functions are modified so that they can be called in parallel (i.e. writing files out to unique names so that a single file isn't being accessed by all 5 cores).

Reply:
Does it completely replace `generate_data.py` by including all the past functionality but also adding this new feature? If so, should it?

Reply:
```generate_data_parallel.py``` does have all the same functionality as ```generate_data.py``` but with the modification to write out to multiple files to accommodate the parallelization as mentioned.  The ```generate_data.py``` should be archived along with the past scripts that were for single simulation runs  -- I can fix that.  The current scripts are running multiple simulations at once and so need the ```generate_data_parallel.py```.